src/controller/raid_floor.controller.ts

```diff
@@ -0,0 +1,115 @@
+import { UpdatedAt } from 'sequelize-typescript';
+import { Floor } from '../models/floor.model.js'
+
+
+///////Read or Pull all
+export const findAllFloor = async (req, res) => {
+    try {
+        console.log("Calling find all for Loot on the Table")
+        const allFloor:Floor[] = await Floor.findAll();
+        res.status(200).json({message: allFloor})
+    } catch (error) {
+       res.status(500).json({ message: error.message });
+    }
+}
+
+//Pull a SINGLE PERSON NEEDS TO BE FIXED
+// export const findSingleLootTable = async (req, res) => {
+//     try {
+//         console.log("Calling find all for Static Mates")
+// //         const allLootTables:Floor[] = await Floor.findAll();
+// //         res.status(200).json({message: allLootTables})
+//     } catch (error) {
+//         res.status(500).json({ message: error.message });
+//     }
+// }
+
+///create
+
+export const createSingleFloor = async (req, res) => {
+    try {
+        //properties on the query object and the query object is property on the request object 
+        const { floor_acronym, floor_full_name } = req.body;
+        console.log("req.body", req.body)
+        //find a single pesron
+        const findSingleFloor = await Floor.findOne({ where: {floor_abbreviation: floor_acronym} });
+        if (findSingleFloor){
+            throw Error("There was already a floor with that name")
+            return
+        }
+        //this is the set up of paramaters for postman
+        const single_floor = await Floor.create({
+                floor_abbreviation: floor_acronym,
+                floor_name: floor_full_name,
+                createdAt: new Date(),
```

Comment:
Remove created_at and updated_at

Reply:
I removed them in raid_floor but noticed that they were in loot_table as well so i removed both o7